backend/inspirehep/accounts/views.py

```diff
@@ -32,12 +32,13 @@ def me():
 
 @blueprint.route("/login-success")
 def login_success():
-    return redirect(session["next_url"])
+    return redirect(session.get("next_url", "/"))
 
 
 @blueprint.route("/signup", methods=["GET"])
 def sign_up_required():
-    return redirect(session["signup_url"])
+    # UI signup form url
+    return redirect("/user/signup")
```

Comment:
at the end, you can remove this view (`/signup`) and add this (`/user/signup`) directly to the config.